ci_framework/roles/dlrn_promote/tasks/get_hash_from_commit.yaml

```diff
@@ -19,36 +21,27 @@
     cifmw_dlrn_promote_distro_hash: "{{ _hashes['commits'][0]['distro_hash'] }}"
     cifmw_dlrn_promote_commit_hash: "{{ _hashes['commits'][0]['commit_hash'] }}"
 
+- name: Converge commit/distro hashes into full_hash
```

Comment:
Maybe this can be merged with https://github.com/openstack-k8s-operators/ci-framework/pull/623/files#diff-9d1804d6713f3765272c790d6878adb4bf05800b9e417a71f4de03e87226eecfR24

Reply:
Done